action/order/routes.go

```diff
@@ -4,8 +4,8 @@ import (
 	"github.com/go-chi/chi"
 )
 
-// Router - Group of order router
-func Router() chi.Router {
+// UserRouter - Group of order router
+func UserRouter() chi.Router {
 	r := chi.NewRouter()
 
 	r.Get("/", list)    // GET /orders - return list of orders
```

Comment:
UserRouter must access * GET /orders/{order_id}* also

Reply:
It is already there at line 15